pddlgym/core.py

```diff
@@ -71,6 +73,7 @@ def get_successor_state(state, action, domain, raise_error_on_invalid_action=Fal
             state,
             effects,
             assignment,
+            get_all_transitions
```

Comment:
What do you think about changing this variable name from "state" to something else? I thought about changing it to "states" but this is not entirely accurate since this might return a single state if the flag is False.
Other possible candidate that I thought of is "result", but this might be too general.

Reply:
Let's leave it as is, so that we don't break dependency code in case they are using keyword arguments when calling this function.